package.json

```diff
@@ -1,37 +1,38 @@
 {
-    "dependencies": {
-        "prettier": "^2.7.1"
-    },
-    "name": "js",
-    "version": "1.0.0",
-    "main": "index.js",
-    "scripts": {
-        "hw16": "npx ts-node ./hw16/index.ts",
-        "hw17-test-mocha": "mocha --config='./hw17/.mocharc.json'",
-        "hw17-test-jest": "jest --config='./hw17/jest.config.cjs'"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/ViktoriyaKhvin/JS.git"
-    },
-    "author": "ViktoriyaKhvin",
-    "license": "ISC",
-    "bugs": {
-        "url": "https://github.com/ViktoriyaKhvin/JS/issues"
-    },
-    "homepage": "https://github.com/ViktoriyaKhvin/JS#readme",
-    "description": "",
-    "devDependencies": {
-        "@types/chai": "^4.3.3",
-        "@types/jest": "^28.1.6",
-        "@types/mocha": "^9.1.1",
-        "@types/node": "^18.7.2",
-        "chai": "^4.3.6",
-        "eslint": "^8.21.0",
-        "jest": "^28.1.3",
-        "mocha": "^10.0.0",
-        "ts-jest": "^28.0.7",
-        "ts-node": "^10.9.1",
-        "typescript": "^4.7.4"
-    }
-}
+
+  "dependencies": {
+    "nyc": "^15.1.0",
+    "save-dev": "^0.0.1-security"
+  },
+  "name": "js",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "hw16": "npx ts-node ./hw16/index.ts",
+    "hw17-test-mocha": "mocha --config='./hw17/.mocharc.json'",
+    "hw17-test-jest": "jest --config='./hw17/jest.config.cjs'",
```

Comment:
Please note that this script is not working. You need to add some missing packages.